deploy/user.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: user.openshift.io/v1
+groups: null
+identities: null
+kind: User
+metadata:
+  name: ${USERNAME}
+---
+apiVersion: user.openshift.io/v1
+kind: Identity
+metadata:
+  name: rhd:${USER_ID_FROM_SUB_CLAIM}
+providerName: rhd
+providerUserName: ${USER_ID_FROM_SUB_CLAIM}
+---
+apiVersion: user.openshift.io/v1
+kind: UserIdentityMapping
+metadata:
+  name: rhd:${USER_ID_FROM_SUB_CLAIM}
+user:
+  name: ${USERNAME}
+identity:
+  name: rhd:${USER_ID_FROM_SUB_CLAIM}
```

Comment:
what is `USER_ID_FROM_SUB_CLAIM` here ??

Reply:
User ID from the access token issued by the RHD Identity Provider. Should be taken from the “sub” claim from the token.
See https://github.com/codeready-toolchain/member-operator/pull/19/files